### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by santokhsingh**
- Application description field
- Bug fix: Approvers were not been able to decline application approval

### CODE REVIEW

File: dao/json/entities/json-applications.js [hunk 1]
```diff
@@ -243,6 +243,9 @@ class JsonApplications {
                         self: { href: '/applications/' + appId }
                     }
                 };
+                
+                if(appCreateInfo.description) //since it's optional
+                    newApp.description = appCreateInfo.description.substring(0, 128);
```
Comment (DonMartin76) [inline dao/json/entities/json-applications.js]: Shouldn't we have more space here, like at least 1K? I can imagine with support for links and things, 128 characters is much too little.

File: dao/postgres/entities/pg-applications.js [hunk 2]
```diff
@@ -120,6 +120,11 @@ class PgApplications {
                 confidential: !!appCreateInfo.confidential,
                 mainUrl: appCreateInfo.mainUrl
             };
+
+            if(appCreateInfo.description){
+              newApp.description = appCreateInfo.description.substring(0, 128);
```
Comment (DonMartin76) [inline dao/postgres/entities/pg-applications.js]: Please create a const on top of this file

`const APP_MAX_LENGTH_DESCRIPTION = 1024;`

or similar

File: routes/applications.js [hunk 3]
```diff
@@ -273,6 +273,9 @@ applications.createApplication = function (app, res, loggedInUserId, appCreateIn
             mainUrl: appCreateInfo.mainUrl
         };
 
+        if(appCreateInfo.description)
+          newAppInfo.description = appCreateInfo.description.substring(0, 128);
```
Comment (DonMartin76) [inline routes/applications.js]: See above, use a constant

File: routes/subscriptions.js [hunk 4]
```diff
@@ -415,7 +415,7 @@ subscriptions.deleteSubscription = function (app, res, applications, loggedInUse
                 return utils.fail(res, 403, 'Not allowed. User invalid.');
 
             var isAllowed = false;
-            if (userInfo.admin)
+            if (userInfo.admin || userInfo.approver)
```
Comment (DonMartin76) [inline routes/subscriptions.js]: This is a bug fix, right?

Comment (santokhsingh) [reply routes/subscriptions.js]: Yes, Approver's were not able to decline the approval

**Pull request merged**